cmd/2019_3/main.go

```diff
@@ -73,18 +61,18 @@ func (g *grid) addWire(wirePath string, index int) {
 
 		switch direction[0] {
 		case 'R': // Right
-			currentStep = g.add(&currentPt, &currentPt.X, step, currentStep, index)
+			currentStep = g.add(currentPt, &currentPt.X, step, currentStep, index)
 		case 'L': // Left
-			currentStep = g.add(&currentPt, &currentPt.X, -step, currentStep, index)
+			currentStep = g.add(currentPt, &currentPt.X, -step, currentStep, index)
 		case 'U': // Up
-			currentStep = g.add(&currentPt, &currentPt.Y, step, currentStep, index)
+			currentStep = g.add(currentPt, &currentPt.Y, step, currentStep, index)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [line is 82 characters, out of limit 80](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498955)